website_logo/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,6 @@
+To configure this module, you need to:
+
```

Comment:
Remove this two lines

Reply:
And I was guided by this.
https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/CONFIGURE.rst
 Should I remove those two lines anyway?